### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by GundlackFelixDEV**
#11 Added Installation Guid
#11 Added Getting Started section with links to Expo and React-Native Tutorials

### CODE REVIEW

**Comment by GundlackFelixDEV** [comment]
@JohannesHilgert  @IvoLeist please review, thanks

**Review by IvoLeist — DISMISSED** [review]
comment I am not entirely convinced of this expo approach yet since we do not know yet with what limitations it comes. How much effort it is going to be to replace the expo dependencies if we realize that we need some components which are not supported by the expo universe...
Nevertheless, I guess for a first start expo should be sufficient.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+   `npm start` or `npm run web`
+
+## Pre-requisities
+- **Node.js**: In order to install Expo CLI you will need to have Node.js (we recommend the latest stable version- but the maintenance and active LTS releases will also work) installed on your computer. [Download the recommended version of Node.js.](https://nodejs.org/en/)
+- **Git**: Additionally, you'll need Git to create new projects. [You can download Git from here.](https://git-scm.com/)
+
+# Getting Started
+1. [**React Native**](https://facebook.github.io/react-native/)
+   > _React Native_ is like _React_, but it uses native components instead of web components as building blocks. 
+So to understand the basic structure of a React Native app, you need to understand some of the basic React concepts, like JSX, components, ``state``, and ``props``.
```
Comment (zappingseb) [inline README.md]: Please describe what JSX is or put a link

**Review by zappingseb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (healthhackersER) [reply README.md]: Done 😀😀

**Review by healthhackersER — APPROVED** [review]
Passt

**Comment by healthhackersER** [comment]
#11 Readme Ready

**Pull request merged**